requirements_no_sql.txt

```diff
@@ -31,3 +31,6 @@ snowballstemmer==1.2.0
 traitlets==4.0.0
 websocket-client==0.34.0
 wsgiref==0.1.2
+corr==0.7.3
+-e git+git://github.com/sma-wideband/adc_tests.git@65a2ef4e1cf68bee35176a1171d923a73952e13e#egg=adc5
```

Comment:
I found a slightly cleaner implementation which installs off the CfA repo and uses the "dependency_links" option.  The install command is now:
`pip install --process-dependency-links ~/test/dragonfly[colorlog,roach]`

On the install, it will claim that that "DEPRECATION: Dependency Links processing has been deprecated and will be removed in a future release." which the internet says is not exactly true.  There is no plan to truly deprecate it because they haven't replaced the functionality yet, but developers want to deprecate it because it has potential security risks.

Tested on zeppelin and confirmed to work.